src/tox/session/commands/show_env.py

```diff
@@ -19,14 +19,17 @@ def report_env(e):
             msg = "{} -> {}".format(e.ljust(max_length), text).strip()
         else:
             msg = e
-        report.line(msg)
+        # Avoiding reporter because that is the result of program execution,
+        # and we do not want to be affected by logging level.
+        print(msg)
```

Comment:
Why does need to change from line to print?

Reply:
If `default environments:` and `additional environments:` use verbosity0, it will be impossible to display environments with descriptions in parsable format.

If you want to list all environments with their descriptions, you must add `-v`, but that also enables those two extra lines above, which render the output as non-parsable. That is caused by another historical mistake of altering the output format based on the verbosity. 

If you want to avoid touching this area we may only fix `--showconfig` and avoid touching the other two. AFAIK, `--showconfig` is a clear case of output that is expected to be machine parseable. The other two, are less important as their main focus was human users. I can live without being able to extra description from `tox -l` or `tox -a` as I am already o parsing --showconfig which already contains it.

Ok to touch only showconfig?

Reply:
My question was more along the lines why are you using print over `report.line` (wasn't talking about verbosity0).

Reply:
> If you want to list all environments with their descriptions, you must add `-v`, but that also enables those two extra lines above, which render the output as non-parsable. That is caused by another historical mistake of altering the output format based on the verbosity.

I don't think this is a mistake. To get all environments you should use `tox -a`. To get just default environments you should use `tox -l`. To get environments with the description you should use `tox --showconfig`.

> AFAIK, `--showconfig` is a clear case of output that is expected to be machine parseable.

Not true. It's actually primarily human-readable, so users can see how their config would be expanded. Secondary to be machine-readable, many of the more complex settings don't have a good machine representation (e.g. enums, etc). Primarly all of these are aimed at human users. Without verbosity `tox -l` and `tox -v` are machine parse-able. tox --showconfig (without verbosity) is intended to be mostly machine parse-able.